### DISCUSSION TITLE
Online/Startup/Offline state of server

### DISCUSSION THREAD

**Discussion by LimitlessDrag0n** [gameserverapp/Platform#434] [Suggestions]
It would be cool to have a widget that tells you the current state your server is in

![image](https://github.com/gameserverapp/Platform/assets/99832504/56ee92c7-017e-4dfd-b029-a6650ea9db3c)

Maybe left of this a indicator for, server is starting up/sever is offline/online with x players

so you do not have to switch to overview to see if your server is actually fully started, fully offline or stuck in some loop etc

**Comment by LimitlessDrag0n** [comment]


**Reply by mvaessen** [reply]
hi, can you please post a new suggestion for this as this seem to be related to other parts of the system.
